#
# glibc-2.2.5 "math test" fix
# (Category - Optional bug fix)
# (Source - glibc cvs HEAD)
#
# glibc's "make check" fails 2 math tests
#
# http://www.zipworld.com.au/%7Egschafer/patches/glibc-2.2.5.fix-math-test.patch.gz
#
diff -uNr glibc-2.2.5.orig/sysdeps/i386/fpu/libm-test-ulps glibc-2.2.5/sysdeps/i386/fpu/libm-test-ulps
--- glibc-2.2.5.orig/sysdeps/i386/fpu/libm-test-ulps	2001-08-27 23:55:13.000000000 +1000
+++ glibc-2.2.5/sysdeps/i386/fpu/libm-test-ulps	2002-07-12 11:44:05.000000000 +1000
@@ -96,9 +96,9 @@
 # cacosh
 Test "Real part of: cacosh (-2 - 3 i) == -1.9833870299165354323470769028940395 + 2.1414491111159960199416055713254211 i":
 double: 1
-float: 4
+float: 9
 idouble: 1
-ifloat: 4
+ifloat: 9
 ildouble: 6
 ldouble: 6
 Test "Imaginary part of: cacosh (-2 - 3 i) == -1.9833870299165354323470769028940395 + 2.1414491111159960199416055713254211 i":
@@ -1234,9 +1234,9 @@
 
 Function: Real part of "cacosh":
 double: 1
-float: 4
+float: 9
 idouble: 1
-ifloat: 4
+ifloat: 9
 ildouble: 328
 ldouble: 328
 
